core/kogitoservice/deployer_test.go

```diff
@@ -167,48 +164,3 @@ func Test_serviceDeployer_DataIndex(t *testing.T) {
 	}
 	test.AssertFetchMustExist(t, cli, topic)
 }
-
-func Test_serviceDeployer_Deploy(t *testing.T) {
```

Comment:
Why is this test removed?

Reply:
This testcase is already covered in `status_test.go`